src/org/mozilla/javascript/ScriptRuntime.java

```diff
@@ -2167,19 +2167,18 @@ public static Object setConst(Scriptable bound, Object value,
      * to see if a given property has already been enumerated.
      *
      */
-    private static class IdEnumeration implements Serializable
-    {
-        private static final long serialVersionUID = 1L;
+    private static class IdEnumeration implements Serializable {
+        private static final long serialVersionUID = 3052885577094294244L;
```

Comment:
Same here -- the class already had a UID so if we change it then we break compatibility.